wherehows-frontend/conf/routes

```diff
@@ -236,6 +236,10 @@ GET           /api/v2/datasets/:urn/retention
 
 POST          /api/v2/datasets/:urn/retention                                   controllers.api.v2.Dataset.updateDatasetRetention(urn: String)
 
+GET           /api/v2/datasets/:urn/exportPolicy                                controllers.api.v2.Dataset.getExportPolicy(urn: String)
```

Comment:
api's url path segments shouldn't be camelCase / case sensitive `/exportpolicy` should be ok